src/common/Utilities/Util.h

```diff
@@ -26,6 +26,13 @@
 #include <utility>
 #include <vector>
 
+enum TimeFormats
```

Comment:
Make this `enum class`, drop TF_ prefix from each value and use CamelCase for naming